package.json

```diff
@@ -70,6 +70,7 @@
     "zone.js": "~0.11.4"
   },
   "dependencies": {
+    "@angular/cdk": "^13.1.1",
```

Comment:
Even if we decide to keep Angular Material, it should be added to the devDependencies. It is only used in the `demo` website and should not be a production dependency of `@fortawesome/angular-fontawesome` NPM package. Otherwise every consumer of the library will end up installing it and unnecessarily bloating their Node modules.